### DISCUSSION TITLE
coc.nvim npm package missing a lot definitions

### DISCUSSION THREAD

**Discussion by jackieli-tes** [neoclide/coc.nvim#3170] [Q&A] [answered]
I'm developing an extension. I'd like to do things like:

- retrive document symbol from language server
- invoke a list

They are available as `import { languages, listManager } from 'coc.nvim'`  but the functions I'm looking for are not there:

```
languages.getDocumentSymbol(document.textDocument, token)
listManager.start(["myownlist"])
```

Looking https://github.com/neoclide/coc.nvim/tree/master/typings, are they deliberately left out? If so, how do I get to these functions?

**Comment by chemzqm** [comment, marked as answer]
To get document symbols, use
```
await nvim.call('CocAction', ['documentSymbols', bufnr])
```
To start the list, use
```
nvim.command('CocList name-of-list', true)
```
would be easier by use viml.
